__std__/nix/plutus/devshells/plutus-shell.nix

```diff
@@ -9,11 +9,27 @@ inputs.std.std.lib.mkShell {
   name = "plutus-shell";
 
   imports = [
+    cell.library.plutus-project.devshell
     inputs.cells.toolchain.devshellProfiles.haskell
   ];
 
-  packages = pkgs.lib.optionals (!pkgs.stdenv.isDarwin) [
+  packages = [
+    pkgs.libsodium-vrf
+    pkgs.libsodium-vrf.dev
+    pkgs.secp256k1
```

Comment:
Why are these necessary? The `haskell.nix` shell should provide them?

Reply:
@jbgi this in fact a little annoying, as `shellFor` would do it automatically.
Apart from this, the new setup seems to work